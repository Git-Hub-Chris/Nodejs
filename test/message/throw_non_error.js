'use strict';

// custom error throwing
throw ({ foo: 'bar' });
