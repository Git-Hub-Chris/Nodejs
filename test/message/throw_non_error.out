*test*message*throw_non_error.js:4
throw ({ foo: 'bar' });
^
[object Object]
